### PR TITLE
chore(eks-cluster): reduce node disk size to 90GB

### PR DESCRIPTION
**Pull request by lemeurherve**
Follow-up of #361 

Tested the size consumed by BOM builds, it consumes 23GB max. Setting the disk size to approximately double this amount should be largely enough and less costly.

Closes https://github.com/jenkins-infra/helpdesk/issues/3423

### CODE REVIEW

File: eks-cluster.tf [hunk 1]
```diff
@@ -96,7 +96,7 @@ module "eks" {
         xvda = {
           device_name = "/dev/xvda"
           ebs = {
-            volume_size           = 200 # Same size as DigitalOcean's size. TODO: Synchronize values between all cloud providers
+            volume_size           = 50
```
Comment (timja) [inline eks-cluster.tf:99]: do we only every run 1 pod per node at the same time?

or is this volume per pod?

Comment (lemeurherve) [reply eks-cluster.tf:99]: You're right, there might be multiple pods running at the same time on the same node, I don't know how much though.

Comment (lemeurherve) [reply eks-cluster.tf:99]: As it's for volumes mounted as `emptyDir`, all of them would compete for the available space on each node, AFAIK there is nothing in that case ensuring they'll get a reserved space.

Comment (dduportal) [reply eks-cluster.tf:99]: As we run 3 pods per nodes, that means we need at least 69 Gb for the emptyDir in current state.
If you had the system space (+ docker images and other emptyDirs), that means ~ 90 Gb is needed.

**Comment by lemeurherve** [comment]
Putting this PR in draft following @timja's comment as I don't know how many BOM pods can run simultaneously on a same node at a given time.
We'll check the consumed space with AWS billing metrics instead of the numbers I've obtained in the BOM build but concerning only one pod at a time.

File: eks-cluster.tf [hunk 2]
```diff
@@ -96,7 +96,7 @@ module "eks" {
         xvda = {
           device_name = "/dev/xvda"
           ebs = {
-            volume_size           = 200 # Same size as DigitalOcean's size. TODO: Synchronize values between all cloud providers
+            volume_size           = 90
```
Comment (dduportal) [inline eks-cluster.tf:99]:
```suggestion
            volume_size           = 90 # With 3 pods / machine, that can use ~25 Gb each at the same time (`emptyDir`)
```

**Review by dduportal — APPROVED** [review]
LGTM

- Terraform plan is ✅ 
- Assuming 23 Gb for a given BOM build, since we have 3 pods running at the same time, it looks good.

**Pull request merged**